tensorflow/compiler/mlir/tosa/transforms/legalize_tfl.cc

```diff
@@ -2998,6 +2999,138 @@ LogicalResult ConvertTFLCosOp::matchAndRewrite(
   return success();
 }
 
+LogicalResult ConvertTFLAtan2Op::matchAndRewrite(
+    Operation* op, PatternRewriter& rewriter) const {
+  auto tfl_atan2_op = cast<TFL::Atan2Op>(op);
+  Location loc = op->getLoc();
+  Value input_y = tfl_atan2_op.getY();
+  RankedTensorType input_y_ty = input_y.getType().dyn_cast<RankedTensorType>();
+  Value input_x = tfl_atan2_op.getX();
+  RankedTensorType input_x_ty = input_x.getType().dyn_cast<RankedTensorType>();
+  ShapedType output_ty =
+      tfl_atan2_op.getResult().getType().dyn_cast<ShapedType>();
+
+  if (!input_y_ty || !input_x_ty) {
+    return rewriter.notifyMatchFailure(
+        op, "ConvertTFLAtan2Op: ranked inputs required");
```

Comment:
Update these to keep it consistent with the others (skipping the pattern name in error message)